pkg/gadgets/tcptracer/tracer/core/bpf/tcptracer.bpf.c

```diff
@@ -0,0 +1,359 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright (c) 2022 Microsoft Corporation
+//
+// Based on tcptracer(8) from BCC by Kinvolk GmbH and
+// tcpconnect(8) by Anton Protopopov
+#include <vmlinux/vmlinux.h>
+
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_core_read.h>
+#include <bpf/bpf_tracing.h>
+#include <bpf/bpf_endian.h>
+#include "tcptracer.h"
+
+const volatile uid_t filter_uid = -1;
+const volatile pid_t filter_pid = 0;
```

Comment:
Why we are not providing to the user the possibility of filtering by PID or UID (this is not even shown to the user)?